index.md

```diff
@@ -76,15 +81,15 @@ Let's add a [Kendo UI Angular DropDownList]({% slug overview_ddl %}) to our app
 ng add @progress/kendo-angular-dropdowns
 ```
 
-2. The `ng add` command will also automatically install the [Default Kendo UI theme]({% slug themesandstyles %}).
+  The `ng add` command will also automatically install the [Default Kendo UI theme]({% slug themesandstyles %}).
```

Comment:
This isn't really a step, so I suggest to add it as an informational sentence outside the numbered list.